test/functional/plugin/lsp/utils_spec.lua

```diff
@@ -0,0 +1,75 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local eq = helpers.eq
+local exec_lua = helpers.exec_lua
+
+describe('vim.lsp.util', function()
+  before_each(helpers.clear)
+
+  describe('stylize_markdown', function()
+    local stylize_markdown = function(content, opts)
+      return exec_lua([[
+        local bufnr = vim.uri_to_bufnr("file:///fake/uri")
+        vim.fn.bufload(bufnr)
+
+        local args = { ... }
+        local content = args[1]
+        local opts = args[2]
+        local stripped_content = vim.lsp.util.stylize_markdown(bufnr, content, opts)
+
+        return stripped_content
+      ]], content, opts)
+    end
+
+    it('should stylize markdown code fences', function()
```

Comment:
"should" is redundant in the test name.

```suggestion
    it('code fences', function()
```